plugins/themes/default/styles/sidebar.less

```diff
@@ -14,7 +14,7 @@
 	font-size: @font-base;
 	line-height: @line-base;
 
-	.title {
+	.title, h2 {
```

Comment:
We should avoid relying on tag-based CSS styles like `h2`, because if the heading structure changes the styles won't be applied. Did you find that the styles were getting overwritten after you changed them to `<h2>`?

Reply:
Yes, what would be the optimal solution in that case?

Reply:
I think if you just figure out which properties are causing problems you can override them in `.title`. For example, I suspect there may be a `margin-top` from the default `<h2>`. In that case, you can override it by adding `margin-top` to `.title`.

Reply:
I've just tested this out myself and it looks like there's just one property, `margin-top`, that's changing when switching between h2 and span. If you add `margin-top: 0` to `.title` that should do the trick.

Reply:
Thanks for catching that! I wasn't sure what had changed.